Core/Src/porting/smsplusgx/main_smsplusgx.c

```diff
@@ -249,8 +249,8 @@ int app_main_smsplusgx(uint8_t load_state)
     memset(audiobuffer_dma, 0, sizeof(audiobuffer_dma));
     HAL_SAI_Transmit_DMA(&hsai_BlockA1, (uint8_t *)audiobuffer_dma, AUDIO_BUFFER_LENGTH_DMA_SMS);
 
-    consoleIsSMS = sms.console == CONSOLE_SMS || sms.console == CONSOLE_SMS2;
-    consoleIsGG  = sms.console == CONSOLE_GG || sms.console == CONSOLE_GGMS;
+    consoleIsSMS = sms.console == CONSOLE_SMS || sms.console == CONSOLE_SMS2 || sms.console == CONSOLE_GGMS;
+    consoleIsGG  = sms.console == CONSOLE_GG;
```

Comment:
This is no longer needed. The render issue was already solved in a different way (the blit() function pointer). The original of these 2 lines were copied verbatim from retro-go and are used for the start/select button mapping.